server/master/src/main/java/org/apache/accumulo/master/replication/RemoveCompleteReplicationRecords.java

```diff
@@ -216,4 +220,13 @@ protected long removeRowIfNecessary(BatchWriter bw, SortedMap<Key,Value> columns
 
     return recordsRemoved;
   }
+
+  /*
+   * Metrics calls snapshot. Largest time stamp will be found for each snapshot then reset.
+   */
+  private void collectLatency(Long createdTime) {
+    long latency = System.currentTimeMillis() - createdTime;
```

Comment:
I am not sure, but I think this createdTime may come from a persisted source?  @joshelser  is that right?  If its coming from a persisted source, then it could have come from another process.  In that case using `System.currentTimeMillis()` is reasonable.   `System.nanoTime()` can reliably compute time deltas within a process.